torch/distributed/_spmd/distribute.py

```diff
@@ -496,11 +496,24 @@ def _convert_to_distributed(
             )
 
         elif isinstance(node.target, torch._ops.OpOverload):
-            replacement = _get_dtensor_dispatch_graph(
-                node, node_to_obj
-            )
-            if replacement is not None:
-                node_replacements[node] = replacement
+            if node.target == torch.ops.aten.scalar_tensor.default:
```

Comment:
TODO for @mrshenli and @wanchaol : add a more generic way to support factory operators.